examples/device/mtp/src/usb_descriptors.c

```diff
@@ -109,18 +109,79 @@ enum
 
 #define CONFIG_TOTAL_LEN (TUD_CONFIG_DESC_LEN + TUD_MTP_DESC_LEN)
 
-uint8_t const desc_fs_configuration[] = {
+// full speed configuration
+const uint8_t desc_fs_configuration[] = {
   // Config number, interface count, string index, total length, attribute, power in mA
   TUD_CONFIG_DESCRIPTOR(1, ITF_NUM_TOTAL, 0, CONFIG_TOTAL_LEN, 0x00, 100),
+  // Interface number, string index, EP event, EP event size, EP event polling, EP Out & EP In address, EP size
   TUD_MTP_DESCRIPTOR(ITF_NUM_MTP, 4, EPNUM_MTP_EVT, 64, 1, EPNUM_MTP_OUT, EPNUM_MTP_IN, 64),
 };
 
+#if TUD_OPT_HIGH_SPEED
+// Per USB specs: high speed capable device must report device_qualifier and other_speed_configuration
+
+// high speed configuration
+uint8_t const desc_hs_configuration[] = {
```

Comment:
```suggestion
const uint8_t desc_hs_configuration[] = {
```

Why not consistent with the order used for the FS configuration above?